### DISCUSSION TITLE
rio_tiler.utils.tile_read import error

### DISCUSSION THREAD

**Discussion by KehanGit** [cogeotiff/rio-tiler#434] [Q&A] [answered]
Hi, 

I am tring to use rio_tiler and I have installed the package following the instructions on the main page. But I had the error as shown below:
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/ubuntu/anaconda3/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/ubuntu/Notebooks/program/SCA_mapping/planet-snowcover/preprocess/__main__.py", line 12, in <module>
    from preprocess import (
  File "/home/ubuntu/Notebooks/program/SCA_mapping/planet-snowcover/preprocess/tile.py", line 12, in <module>
    from rio_tiler.utils import tile_read
ImportError: cannot import name 'tile_read' from 'rio_tiler.utils' (/home/ubuntu/Notebooks/program/SCA_mapping/rio-tiler/rio_tiler/utils.py)

Can you help me with this? Thank you!

**Comment by kylebarron** [comment, marked as answer]
> on the main page

What main page? `rio_tiler.utils.tile_read` existed in an old version of `rio-tiler` (v1) but not in the current 2.x version. I don't see any references to `tile_read` in either the README.md or any part of the documentation.

**Reply by KehanGit** [reply]
I installed the package following:
$ git clone https://github.com/cogeotiff/rio-tiler.git
$ cd rio-tiler
$ pip install -U pip
$ pip install -e .

Thank you for your clarification. I did not find it either. Is there a way to install the old version?

**Reply by kylebarron** [reply]
You can install more simply with just `pip install rio-tiler`. You can force pip to install a version `<2` but in most cases it would make more sense just to use version 2's API.
